srcopsmetrics/github_knowledge.py

```diff
@@ -72,6 +72,28 @@ def connect_to_source(self, project: Tuple[str, str]) -> Repository:
 
         return repo
 
+    @staticmethod
+    def get_repositories(repository: str = None, organization: str = None) -> List[str]:
```

Comment:
We should have a warning if no repos has been identified, WDYT?

Reply:
we should have warning if organization contains zero repositories
one repository would always be found, if not - an exception from pygithub occurs that repository does not exist
hence the warning should be only for organizations I think

Reply:
Perfect! Thanks :)